src/Data/Table/Base.agda

```diff
@@ -6,46 +6,59 @@
 
 module Data.Table.Base where
 
+open import Data.Nat
 open import Data.Fin
+open import Data.Product using (_×_ ; _,_)
 open import Data.List as List using (List)
-open import Data.Nat
 open import Function using (_∘_)
 
 record Table {a} (A : Set a) n : Set a where
```

Comment:
I would prefer the `{n}` parameter to come first, as that's the one that needs to be specified most frequently.